src/backend/distributed/citus--7.0-9--7.0-10.sql

```diff
@@ -0,0 +1,27 @@
+/* citus-7.0-9--7.0-10 */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION master_add_secondary_node(nodename text,
+                                          nodeport integer,
+                                          primaryname text,
+                                          primaryport integer,
+                                          nodecluster name default 'default',
+                                          OUT nodeid integer,
+                                          OUT groupid integer,
+                                          OUT nodename text,
+                                          OUT nodeport integer,
+                                          OUT noderack text,
+                                          OUT hasmetadata boolean,
+                                          OUT isactive bool,
+                                          OUT noderole noderole,
+                                          OUT nodecluster name)
```

Comment:
In this case it turned out to be kind of useful, it made testing a little easier but I think you're right both that we should return a boolean, whether the node was added (and didn't already exist) and that the change should be a single PR which changes them all at once.